src/forms/TogglePill.jsx

```diff
@@ -97,7 +91,7 @@ class TogglePill extends React.PureComponent {
 							className={labelClassNames}
 							htmlFor={id}>
 							{children}
-							{(topic) ? topicChildren : null}
+							{(topic) ? topicChildren(isActive) : (isActive) => null}
```

Comment:
I don't think the function in the right hand return here is doing anything. You could just do
`{(topic) ? topicChildren(isActive) : null}`